decentraland/unity-explorer
# Fix: changed visuals of system messages

## What does this PR change?

Fix #743 adds new visuals for system messages in chat similar to the following design:

<img width="371" alt="Screenshot 2024-05-07 at 22 52 48" src="https://github.com/decentraland/unity-explorer/assets/4254522/2bf4512c-b19a-4c18-8b0d-89c43fc59c91">


...

## How to test the changes?



1. Launch the explorer
2. type /goto 0,0
3. Check that the system message in chat is different from own message
4. Check that there is no chat bubble above us with the system message

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
